cmake/Setup.cmake

```diff
@@ -1,9 +1,9 @@
-######################################################################################
-# Copyright 2024 Lawrence Livermore National Security, LLC and other CARE developers.
-# See the top-level LICENSE file for details.
+##############################################################################
+# Copyright (c) 2024-24, Lawrence Livermore National Security, LLC and CARE
```

Comment:
Do you care (sorry, no pun intended) that this says "2024-24"?

Reply:
Fixed.